### PR TITLE
CTCTRADERS-451: Updated stubs to send rejection response

### PR DESCRIPTION
**Pull request by MatthewAlexanderBeck**

### CODE REVIEW

File: app/controllers/FakeResponseController.scala [hunk 1]
```diff
@@ -28,78 +30,93 @@ import renderer.Renderer
...
 import uk.gov.hmrc.viewmodels.NunjucksSupport
 
+import scala.concurrent.ExecutionContext
 import scala.xml.Elem
 import scala.xml.{XML => xmlFile}
-import scala.concurrent.ExecutionContext
-import scala.concurrent.Future
 
-class ResponseController @Inject()(
+class FakeResponseController @Inject()(
```
Comment (ashwinirudrappa) [inline app/controllers/FakeResponseController.scala:37]: @MatthewAlexanderBeck the name FakeResponseController feels quite not right,  how about naming it as NCTSResponseController or something in similar lines

File: conf/app.routes [hunk 2]
```diff
@@ -1,10 +1,10 @@
 # microservice specific routes
 
-GET        /assets/*file                                     controllers.Assets.versioned(path="/public", file: Asset)
-GET        /language/:lang                                   controllers.LanguageSwitchController.switchToLanguage(lang: String)
+GET                     /assets/*file                           controllers.Assets.versioned(path="/public", file: Asset)
+GET                     /language/:lang                         controllers.LanguageSwitchController.switchToLanguage(lang: String)
```
Comment (ashwinirudrappa) [inline conf/app.routes:4]: we can remove language switch if it does not break anything working

File: app/controllers/FakeResponseController.scala (hunk 1 above)
Comment (MatthewAlexanderBeck) [reply app/controllers/FakeResponseController.scala:37]: @ashwinirudrappa hey Ashwini, do you think EISResponse would also be suitable as that is the end point we are try to mimic?

File: app/controllers/EISResponseController.scala [hunk 3]
```diff
@@ -28,78 +30,93 @@ import renderer.Renderer
...
-import scala.concurrent.ExecutionContext
-import scala.concurrent.Future
 
-class ResponseController @Inject()(
+class EISResponseController @Inject()(
   override val messagesApi: MessagesApi,
   cc: MessagesControllerComponents,
   destinationConnector: DestinationConnector,
-  formProvider: ResponseForm,
+  formProvider: FakeResponseForm,
```
Comment (ashwinirudrappa) [inline app/controllers/EISResponseController.scala:41]: @MatthewAlexanderBeck form still says FakeResponseForm :)

File: app/models/FakeResponse.scala [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.libs.json.Json
+import play.api.libs.json.OFormat
+
+case class FakeResponse(arrivalId: String, messageCorrelationId: String, messageType: String)
```
Comment (ashwinirudrappa) [inline app/models/FakeResponse.scala:22]: @MatthewAlexanderBeck one more

**Pull request merged**